crud/common/utils.lua

```diff
@@ -524,4 +601,10 @@ function utils.flatten_obj_reload(space_name, obj)
     return schema.wrap_func_reload(flatten_obj, space_name, obj)
 end
 
+function utils.fetch_ddl_sharding_key(conn)
```

Comment:
I don't understand why do you add such one-line function and can't use it directly from schema module

Reply:
AFAIR there is a loop in dependencies and I have added a wrapper for `fetch_ddl_sharding_key()` to `utils`.